src/retyped/flowprinter.re

```diff
@@ -32,9 +29,14 @@ let rec show_type =
                 | _ => name ^ ": " ^ show_type type_of
                 }
             )
-            params
-        ) ^
-      "): " ^ show_type return
+            params;
```

Comment:
If you use the native compiled binary in debug mode like `./_build/default/src/cli.exe -debug _test.js` it will print out how it interpreted the JS code

Good way to confirm you're parsing correctly